tket/test/src/Circuit/test_Circ.cpp

```diff
@@ -1020,6 +1020,29 @@ SCENARIO(
   test.assert_valid();
 }
 
+SCENARIO(
+    "Test a circuit with blank wires can have the blank wires removed keeping "
+    "classical",
+    "[blank_wires]") {
+  Circuit test(4, 2);
+  test.add_op<unsigned>(OpType::CX, {0, 1});
+  test.add_op<unsigned>(OpType::Z, {0});
+
+  WHEN("Check Commands work correctly") {
+    std::vector<Command> coms = test.get_commands();
+    REQUIRE(*coms[0].get_op_ptr() == *get_op_ptr(OpType::CX));
+    REQUIRE(*coms[1].get_op_ptr() == *get_op_ptr(OpType::Z));
+  }
+
+  test.add_blank_wires(8);
+  int n = test.n_vertices();
+  test.remove_blank_wires(true);
+  int m = test.n_vertices();
+  REQUIRE(n == 30);
+  REQUIRE(m == 10);
+  test.assert_valid();
```

Comment:
I think a test that just checked the number of qubits and bits in the circuit rather than the number of vertices would be clearer, but no big deal.

Reply:
I just copied that from the old testcase, happy to update both.

Reply:
done in [7f09164](https://github.com/CQCL/tket/pull/1435/commits/7f091640741c1fa94badc14c57602ef4e4d0279e)